rma/models/rma.py

```diff
@@ -1261,6 +1258,26 @@ def _delivery_group_key(self):
         self.ensure_one()
         return (self.partner_shipping_id.id, self.company_id.id, self.warehouse_id.id)
 
+    def _reception_group_key(self):
```

Comment:
I think calling methods by this convention makes better to understand its goal (`get`: return something):

```suggestion
    def _get_reception_group_key(self):
```

Reply:
This comment is still pending.

Reply:
Agree, but I wanted to follow the same naming for delivery

https://github.com/OCA/rma/blob/e9e639f6fef7bd5069e80e8ba9d4016098b10c15/rma/models/rma.py#L1271-L1274

Reply:
Well, if something is incorrect previously, let's do a `[REF]` changing it. We can keep the previous method name calling the new one for compatibility.

Reply:
Ok I will do,

Reply:
done